aiographite/protocol.py

```diff
@@ -0,0 +1,73 @@
+from typing import Dict, Tuple, List
+import pickle
+import struct
+
+
+class PlaintextProtocol:
+
+	def data_format(self, metric: str, value: int, timestamp: int) -> str:
```

Comment:
how about format_data as the name of function? by the name it sounds its performing formatting of data. data_format sounds more like a property. or may be it's just me :)